src/main/java/com/bynder/sdk/util/Utils.java

```diff
@@ -50,4 +52,20 @@ public static boolean isDateExpiring(final Date date, final int seconds) {
         expirationDate.add(Calendar.SECOND, -seconds);
         return expirationDate.before(Calendar.getInstance());
     }
+
+    /**
+     * Loads a (.properties) config file from the resources.
+     *
+     * @param file the filename of the config file (without the extension)
+     * @return {@link Properties} with the values from the config file
+     * @throws IOException if the config file could not be found
+     */
+    public static Properties loadConfig(final String file)
```

Comment:
This deserves a unit test

Reply:
Thought of that, but what would we want to test exactly though? There's no real logic here, except stuff that's handled by other classes, like loading a .properties file into a Properties object.

Reply:
You're offering a way to load a .properties file from a specific location. It makes sense to have that verified in a unit test. Could be as simple as a .properties file in `/src/test/resources/com/bynder/sdkutil/` that you try to read. Or maybe should it be `/src/test/resources/`? I always forget the exact location.

Reply:
Added some unit tests for this method.